### DISCUSSION TITLE
Use TMDB aliases for Scene Exceptions

### DISCUSSION THREAD

**Discussion by reconman** [pymedusa/Medusa#9725] [Ideas & Feature requests]
TVDB has XEM mappings for Scene Exceptions, but TMDB does not have XEM mappings.

Therefore I would propose using the TMDB alias list for generating the Scene Exceptions. Otherwise, users have to add the Japanese title to every TMDB anime.

**Comment by p0psicles** [comment]
You can already add scene exceptions for tmdb and tvmaze.

**Reply by reconman** [reply]
Like I said

> Otherwise, users have to add the Japanese title to every TMDB anime.

**Reply by p0psicles** [reply]
https://github.com/pymedusa/medusa.github.io/blob/master/scene_exceptions/scene_exceptions_tmdb.json

Everyone can add new exceptions to this file.

**Reply by reconman** [reply]
That's not good enough, you really expect people to update that file every season even though TMDB is already filled with the same info?

**Reply by p0psicles** [reply]
Sorry, what do you mean. Does tmdb keep scene exceptions?

**Reply by reconman** [reply]
Sure it does, for example:
- https://www.themoviedb.org/tv/111819/titles?language=en-US
- https://www.themoviedb.org/tv/1429/titles?language=en-US

**Reply by p0psicles** [reply]
I didn't know about that.
But I see they have a lot. But it's kinda messy. Like some season exceptions are marked as season but others are not.
for example look at jojo's bizarre adventure

So the questions that need to be answered here are.
1. Which languages / type combinations should we use.
2. Should we support alternative language when the shows base language is alternative. Like a french show, should we allow french scene exceptions (and the default)
3. how would be handle the language when user adds the shows in a specific language?

We should not add each language. Cause every language results in *2 or 3 in search results.

If tmdb would also have airtime, they would be the perfect replacement for tvdb.

**Reply by reconman** [reply]
Speaking of, do you know of the incoming TVDB doomsday of October 1st where the old APIs are shut down and you have to pay in order to use the API?

https://www.facebook.com/TheTVDB/posts/10159183527546041

**Reply by p0psicles** [reply]
Yeah didn't know the date. Tnx
